crates/extensions/c8y_mapper_ext/src/error.rs

```diff
@@ -75,6 +75,9 @@ pub enum ConversionError {
     #[error("Unsupported topic: {0}")]
     UnsupportedTopic(String),
 
+    #[error(transparent)]
+    MqttSchemaError(#[from] tedge_api::mqtt_topics::EntityTopicError),
```

Comment:
Looks like this isn't used, I removed this variant and the code still compiles.

Reply:
Fixed: https://github.com/thin-edge/thin-edge.io/pull/2189/commits/dd086f22ef2597478b3f0a7a2392e8a6e6fac00a